cmd/ris-mirror/rismirror/router.go

```diff
@@ -12,10 +13,12 @@ import (
 
 // Router represents a router
 type Router struct {
-	name        string
-	address     net.IP
-	vrfRegistry *vrf.VRFRegistry
-	vrfs        map[uint64]*_vrf
+	name         string
+	address      net.IP
+	vrfRegistry  *vrf.VRFRegistry
+	vrfs         map[uint64]*_vrf
+	vrfClients   map[uint64][]*risclient.RISClient
```

Comment:
I'm wondering why we don't maintain a VRFs clients within the `_vrf` object? Would be more straight forward I guess.

Reply:
Done. I was a bit careful about that, because of the relation to `vrf.VRF`.